controllers/authorController.ts

```diff
@@ -0,0 +1,9 @@
+var Author = require('../models/Author');
+
+exports.authors_list = function(req, res) {
+    res.send('NOT IMPLEMENTED: Author list');
+};
+
+exports.author_songs = function(req, res) {
+    res.send('NOT IMPLEMENTED: Author detail: ' + req.params.id);
+};
```

Comment:
![image](https://user-images.githubusercontent.com/64740665/104043407-d275cf80-51ec-11eb-8a42-ecba6d47b607.png)
 You should use Typecript typing!